Project/Fall2020_CSC403_Project/TextEngine.cs

```diff
@@ -49,32 +48,98 @@ public void TextboxVisible(bool hide)
             Textbox.Visible = hide;
         }
 
-        private void Textbox_Click(object sender, EventArgs e)
-        {
-            string line = story.GetNextLine();
-            this.ChangeText(line);
-        }
-
-        private void TextEnginge_KeyPress(object sender, KeyPressEventArgs e)
+        private void TextEngine_KeyPress(object sender, KeyEventArgs e)
         {
-            string line = story.GetNextLine();
-            this.ChangeText(line);
+            if (this.Options.Count != 0)
+            {                
+                if(e.KeyCode == Keys.Down && Options.Count > 1)
```

Comment:
i know you're a fan of switch statements, you could probably use one here? maybe?